### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by amcsi**
Cool project you have.

I helped Dockerize it for you.

I'd suggest you create an account on https://hub.docker.com/ for creating automated builds of the project, so users wouldn't have to build the project themselves. Also you could add yourself as the MAINTAINER of the Dockerfile which I left out, because I don't know your name/email ( like what I did in my project: https://github.com/amcsi/szeremi/blob/master/Dockerfile#L2 ).

### CODE REVIEW

**Comment by danistefanovic** [comment]
Hey Attila, thanks for your contribution! It works like a charm. Found just some minor improvements. Other than that, I think this is good to go. 👍

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Dockerfile
+.git
+node_modules
+npm-debug.log*
+webhooks.json
```
Comment (danistefanovic) [inline .dockerignore]: * [ ] Add `lib` folder to .dockerignore, which is generated when you build the project locally

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM node
```
Comment (danistefanovic) [inline Dockerfile]: * [ ] Add tag of the latest LTS version 
```
FROM node:4.4
```

**Comment by amcsi** [comment]
Understood. I have made the commit with what you said and it seems to still work.

I'm still investigating how you can control Docker from within a container. I want to use your project to auto-pull and restart Docker containers with the help of Docker Hub webhooks, but I want to run hooka from within a container, because my server uses CoreOS where everything has to be in a container.

**Comment by amcsi** [comment]
I was able to get `docker ps` to run within the container this way: `docker run --rm -v ~/hooka/webhooks.json:/src/webhooks.json -p 3000:3000 --name hooka -v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker -v /lib/x86_64-linux-gnu:/lib/x86_64-linux-gnu -v /usr/lib/x86_64-linux-gnu/:/usr/lib/x86_64-linux-gnu/ hooka docker ps`

I'm concerned of the last two lib mounts might be platform dependent, so I'm investigating a better way.

**Comment by danistefanovic** [comment]
I think there is no one-size-fits-all solution. I had to do this acrobatics to make it work under OSX with boot2docker:
```
docker run -v ~/Workspace/hooka/webhooks.json:/src/webhooks.json -v /usr/local/bin/docker:/usr/bin/docker -v $DOCKER_CERT_PATH:/certs -e "DOCKER_HOST=$DOCKER_HOST" -e "DOCKER_CERT_PATH=/certs" -e "DOCKER_TLS_VERIFY=1" hooka docker ps
```

**Comment by amcsi** [comment]
Oh crap i didnt realize i committed that change that removes the Expose command from the dockerfile. I think that command may be needed for things like random port binding to work

**Comment by danistefanovic** [comment]
No problem. I've fixed it d53b71934b4f04e6b484f102df01637911bdb4ff

**Pull request merged**